CMakeLists.txt

```diff
@@ -6,6 +6,12 @@ OPTION(USE_HDFS "Enable HDFS support (EXPERIMENTAL)" OFF)
 OPTION(USE_TIMETAG "Set to ON to output time costs" OFF)
 OPTION(USE_CUDA "Enable CUDA-accelerated training (EXPERIMENTAL)" OFF)
 OPTION(USE_DEBUG "Set to ON for Debug mode" OFF)
+OPTION(USE_SANITIZER "Use santizer flags" OFF)
+OPTION(SANITIZER_PATH "Path to sanitizer libs")
```

Comment:
Based on https://github.com/microsoft/LightGBM/pull/3555#discussion_r588814820, this should be `SET`, right?